packages/twenty-server/src/database/typeorm/core/core.datasource.ts

```diff
@@ -18,9 +18,15 @@ export const typeORMCoreModuleOptions: TypeOrmModuleOptions = {
   migrationsRun: false,
   migrationsTableName: '_typeorm_migrations',
   metadataTableName: '_typeorm_generated_columns_and_materialized_views',
-  migrations: [
-    `${isJest ? '' : 'dist/'}src/database/typeorm/core/migrations/*{.ts,.js}`,
-  ],
+  migrations:
+    process.env.IS_BILLING_ENABLED === 'true'
+      ? [
```

Comment:
I think this works but I'm not sure how it will behave between migrate up and down in case we change IS_BILLING_ENABLED value. Let's say it is not meant to be changed :p